src/main/resources/config/liquibase/changelog/20231120095701_changelog.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="andreas (generated)" id="1700470627638-3">
+        <createTable tableName="math_exercise_details">
+            <column name="example_solution" type="VARCHAR(255)"/>
+            <column name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="math_exercise_detailsPK"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="andreas (generated)" id="1700470627638-4">
+        <createTable tableName="math_submission_details">
+            <column name="language" type="VARCHAR(255)"/>
```

Comment:
255 is bigger than it needs to be. Elsewhere in Artemis 10 and 25 are also used. (This should probably be unified in the future)
<img width="319" alt="image" src="https://github.com/ls1intum/Artemis/assets/18218285/f1f44ee9-2ffd-43ec-b849-88975e862c95">